### DISCUSSION TITLE
Is there problem for doing many tasks with very long delay

### DISCUSSION THREAD

**Discussion by ragokan** [taskforcesh/bullmq#1157] [Q&A] [answered]
Hello, for my app I have to use delayed tasks but the delay starts from a week to 4 months (per user, at random dates). I searched about it on Internet and for this purpose Bull seemed to be the best one. Yet I just wanted to ask if it is not suitable for Bull to have delays for longer than 3 months. The operation goes like this:

User creates an operation with reminder, lets say 3 days. After 3 days, I want to execute remind function for user, so that I can do operation and send notification to that user.

**Comment by manast** [comment, marked as answer]
There are users using delays longer than a year.

**Reply by ragokan** [reply]
Ahh, really good to hear. I hope there is no problem of having many tasks, too (like 200k/monthly). Thank you so much for your reply!

**Reply by manast** [reply]
200k monthly is nothing...

**Reply by manast** [reply]
Just make sure that you have enough space in Redis and/or that you only keep a handful of completed jobs to keep memory low.

**Reply by ragokan** [reply]
Thank you so much, it really helped a lot!
